tests/unit/backend/io/test_metadata.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""
+Unit tests for backend metadata auxiliary class.
+"""
+import pytest
+import numpy as np
+import tifffile
+from imars3d.backend.io.metadata import MetaData
+from imars3d.backend.io.metadata import _extract_metadata_from_tiff
+
+
+@pytest.fixture(scope="module")
+def test_data(tmpdir_factory):
```

Comment:
Because of the the name starting with `test_data`, pytest may think this is a test. Can you change the name to `make_test_data` or `data_fixture`?

Reply:
name updated